backend/apps/pr_review_bot/models.py

```
@@ -0,0 +1,181 @@
+        ('critical', 'Critical'),
+    ]
+    
+    TYPE_CHOICES = [
+        ('style', 'Style'),
+        ('complexity', 'Complexity'),
+        ('duplication', 'Duplication'),
+        ('security', 'Security'),
+        ('test', 'Test'),
+        ('performance', 'Performance'),
+        ('maintainability', 'Maintainability'),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)